data/queenstack.json)

```diff
@@ -0,0 +1,22 @@
+
+ { "name": "Agada Queen",
+  "type": "personal",
+  "bio": "Data scientist **Business Analyst**. Technical writer"
+  "links": [
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/maquinbeaury",
+       "icon": "FaTwitter"
+       }
+       {
+      "name": "Follow me on Linkedin",
+      "url": "https://twitter.com/maquinbeaury",
```

Comment:
You have given twitter link in here, instead of your linkedin profile , please change it......